Update define-command-argument-container.md (#45900)
kubernetes/website: content/en/docs/tasks/inject-data-application/define-command-argument-container.md

with your new arguments.

{{< note >}}

> The `command` field corresponds to `ENTRYPOINT`, `args` field corresponds to `CMD` in some container runtimes.

Reviewer: I would suggest a small grammatical change

```suggestion
The `command` field corresponds to `ENTRYPOINT`, and the `args` field corresponds to `CMD` when `ENTRYPOINT` is used.
```

Author: I remain [and the] to fix grammatical error. and use [in some container runtimes] before.
